scripts/push-documentation.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+setup_git() {
+  git config --global user.email "nakul.korde@intel.com"
+  git config --global user.name "Nakul Korde"
+}
```

Comment:
Should we set up a `travis` user? Just a thought...

Reply:
That is good idea although the credentials are encrypted, might be better to have a dedicated travis account for future use; will be beneficial for management.